src/ProgramBlockEditor.scss

```diff
@@ -199,17 +195,27 @@ button.btn.ProgramBlockEditor__program-deleteAll-button--pressed.ProgramBlockEdi
         fill: #4C9990;
         stroke: none;
     }
+
+    .ProgramBlockEditor__character-position-button--disabled {
+        path.filled {
+            fill: #818A98;
+        }
+    }
 }
 
 .ProgramBlockEditor__character-position-button:focus {
     outline: $c2lc-focus-indicator-width solid $c2lc-focus-indicator-color;
 }
 
-.ProgramBlockEditor__character-position-button--disabled {
-    stroke: $c2lc-disabled-foreground-color;
+.ProgramBlockEditor__character-position-button--disabled,
+.ProgramBlockEditor__character-turn-positions--disabled {
```

Comment:
I think that this `.ProgramBlockEditor__character-turn-positions--disabled` selector isn't necessary. As I don't think that this class is ever set on an element. The individual turn buttons will have `ProgramBlockEditor__character-position-button--disabled`, but the container for the turn buttons will only have `ProgramBlockEditor__character-turn-positions`, without the `--disabled`.

Reply:
Verified, and fixed.